src/main/java/de/dhbw/karlsruhe/domain/models/SudokuGeneratorTransformation.java

```diff
@@ -0,0 +1,413 @@
+package de.dhbw.karlsruhe.domain.models;
+
+import java.util.*;
+
+public class SudokuGeneratorTransformation {
```

Comment:
Klasse ist an sich mit über 400 Zeilen zu groß. Eine weitere Aufteilung in Klassen wäre sinnvoll, bspw. in:
- Transform Klasse
- Removing Klasse
- Solver Klasse
- Validation Klasse (ist ja in #9 angedacht)

Reply:
Ich habe eine Remover Klasse erstellt. Soll ich auch SudokuGeneratorBacktracking so anpassen, dass diese neue Klasse verwendet wird?

Reply:
Die Transformator-Klasse, die ich erstellt habe hat immer noch ca. 208 Zeilen, soll ich die weiter Aufteilen oder nehmen wir die als negativ Beispiel. Thematisch denke ich ist eine Aufteilung auch nicht wirklich besser zu verstehen als, wie es jetzt ist.

Reply:
Backtracking anpassung wäre natürlich super :D kann ich aber auch später noch machen. Denke die Transformator-Klasse ist so ok.